tests/manager/api/test_opds_for_distributors.py

```diff
@@ -526,40 +528,54 @@ def test_import(self, opds_dist_api_fixture: OPDSForDistributorsAPIFixture):
         # it adds delivery mechanisms for books with epub acquisition links
         # and sets pools' licenses_owned and licenses_available.
 
-        # Both works were created, since we can use their acquisition links
+        # All four works in the feed were created, since we can use their acquisition links
```

Comment:
Going from "Both" to "All four" by adding one entry confused me for a minute.  🤦🏽‍♂️